about.html

```diff
@@ -0,0 +1,114 @@
+                </div>
+            </nav>
+            
+            <div class="grade"></div>
+            <div class="section-black">
+              <section id="projts">
+                <h2>Courses Offer</h2>
+                <article>
+                  <div class="textz">
+                    <h4>Project Realise</h4>
+                    <h3>Wall of Wonder</h3>
+                  <p class="blackbox"> Throughout this program, you will develop skill sets in a variety of technologies including Cloud foundations, HTML, CSS, JavaScript, GitHub, Node.js, React, Cloud Native practices, DevOps, CI/CD, Containers, Docker,.  <a href="https://frontendmasters.github.io/grid-flexbox-v2/">this</a>, and multiple links look <a href="#">like this</a> and <a href="#">like this</a>.</p>
+                    <h4>Technologies used include:</h4>
+                    <ul>
+                      <li>HTML</li>
+                      <li>CSS</li>
+                      <li>SVG</li>
+                    </ul>
+                  </div>
+                    <img src="https://assets.codepen.io/296057/fem-gettingstartedcss-ch5-1.png" alt="Screenshot of the Wall of Wonder." />
+                </article>
+                
+            
+            
+                <article class="reverse">
+                  <div class="text">
+                    <h4>Bootcamp Project</h4>
+                    <h3>Capston Projects</h3>
+                    <p class="blackbox">
+                      Description of the project. This should be fairly concise while also describing the key components that you developed or worked on. It can be as long as you need it to be but should at least be a few sentences long. Be sure to include specific links anywhere
+                      in the description. A link looks like
+                      <a href="https://citec-47.github.io/capston-project/about.html">this</a>, and multiple links look <a href="#">like this</a> and
+                      <a href="#">like this</a>.
+                    </p>
+                    <h4>Technologies used include:</h4>
+                    <ul>
+                      <li>HTML</li>
+                      <li>CSS</li>
+                      <li>JavaScript</li>
+                    </ul>
+                  </div>
+                  <img src="https://assets.codepen.io/296057/fem-gettingstartedcss-ch5-5.png" alt="Screenshot of the Frontend Masters Bootcamp." />
+                </article>
+              </section>
+              </div>
+              
+              
+  <div class="grade"></div>
+  <div class="cont">
+  <section id="contact">
+    <h2>Contact Us</h2>
+  
+  <p>We are always interested in hearing from you. we wl'd like to get emails and contacted about everything
+    </p>
+  
+    <p><a href="mailto:mauricendonyi40@gmail.com" class="button">Email Us</a>
+    </section>
+    <div class="grade"></div>
+    </div>
+    <div class="gradient"></div>
+    <footer>
```

Comment:
According to the design, before the footer, there should exist two sections that has:

- A header with a decorative underline plus a paragraph of text, and an image or large logo. see below

![bbbbbad](https://user-images.githubusercontent.com/84930355/225390368-617d6b1e-7ebb-498f-aa68-dfda0d98c37f.PNG)

- a section with a centered column that includes a section header, a divider, a line of text, and two images with text centered over them. see the image below

![Am](https://user-images.githubusercontent.com/84930355/225391243-0a899127-259c-42b5-a13b-e95d0efc1b50.PNG)

Kindly address all the above mentioned to make your design similar to that of the behance version